packages/graph/hash_graph/lib/graph/src/store/crud.rs

```diff
@@ -28,7 +28,7 @@ pub trait Read<T: Send>: Sync {
     /// Returns a value from the [`Store`] specified by the passed `query`.
     ///
     /// [`Store`]: crate::store::Store
-    async fn read<'query>(&self, query: &Self::Query<'query>) -> Result<Vec<T>, QueryError>;
+    async fn read<'f: 'q, 'q>(&self, query: &'f Self::Query<'q>) -> Result<Vec<T>, QueryError>;
```

Comment:
(comment for line 23 here).
Is a `Valuable` trait still something we want? There's also reference to it in `api/rest/mod.rs:102`

Reply:
It is this https://docs.rs/valuable/latest/valuable/trait.Valuable.html, right? Perhaps it would be useful to link to it with rustdocs :)

Reply:
Good point, I added a link to the trait in 2f9fa60f6fc25cd12d19cb7dfb66c0f373343a0d